subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -894,10 +894,11 @@ static int prepare_cb(struct lll_prepare_param *p)
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
 	/* TODO: if coded we use S8? */
 	radio_phy_set(lll->phy_p, lll->phy_flags);
-	radio_pkt_configure(8, PDU_AC_LEG_PAYLOAD_SIZE_MAX, (lll->phy_p << 1));
+	radio_pkt_configure(8, PDU_AC_LEG_PAYLOAD_SIZE_MAX, RADIO_PKT_CONF_PHY(lll->phy_p));
```

Comment:
Should this also have been
```suggestion
	radio_pkt_configure(RADIO_PKT_CONF_LENGTH_8BIT, PDU_AC_LEG_PAYLOAD_SIZE_MAX, RADIO_PKT_CONF_PHY(lll->phy_p));
```
?

Reply:
Corrected, thanks for catching this!